pygments/lexers/prolog.py

```diff
@@ -244,44 +246,43 @@ class LogtalkLexer(RegexLexer):
         'directive': [
             # Conditional compilation directives
             (r'(el)?if(?=[(])', Keyword, 'root'),
-            (r'(e(lse|ndif))[.]', Keyword, 'root'),
+            (r'(e(lse|ndif))(?=[.])', Keyword, 'root'),
```

Comment:
If a dot always follows, can it be highlighted in this rule?

Reply:
As you may know, Logtalk (and Prolog) source files are made of *terms* (directives and clauses; clauses can be facts or rules) that must always end with a dot. Thus, consistent highlighting would require to all ending dots be highlighted, not just in this particular case.